x-pack/plugins/session_view/public/components/TestPage/index.tsx

```diff
@@ -21,7 +21,7 @@ import { RouteComponentProps } from 'react-router-dom';
 import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
 import { CoreStart } from '../../../../../../src/core/public';
 import { BASE_PATH, INTERNAL_TEST_ROUTE } from '../../../common/constants';
-import SessionView from '../SessionView';
+import { SessionView } from '../SessionView';
```

Comment:
Is exporting default not allowed?

Reply:
it's not, also an Eslint rule (which I find to be very opinionated, but ok, at least makes it consistent)